test/controllers/tasks_controller_test.rb

```diff
@@ -0,0 +1,118 @@
+require "test_helper"
+
+describe TasksController do
+  let (:task) {
+    Task.create name: "sample task", description: "this is an example for a test",
+                completion_date: Time.now + 5.days
+  }
+
+  # Tests for Wave 1
+  describe "index" do
+    it "can get the index path" do
+      # Act
+      get tasks_path
+
+      # Assert
+      must_respond_with :success
+    end
+
+    it "can get the root path" do
+      # Act
+      get root_path
+
+      # Assert
+      must_respond_with :success
+    end
+  end
+
+  # Uncomment these tests for Wave 2
+  # describe "show" do
```

Comment:
Should these be `xdescribe`s instead of commented out?

Reply:
Apologies, I hadn't loaded minitest-skip in my template.

Reply:
And now I remember, minitest-skip is currently not compatible with minitest-rails...